### ISSUE TITLE
Challenges:' View Participant Progress' text is white instead of dark gray

### ISSUE THREAD

**Issue by saraolson** [HabitRPG/habitrpg-ios#582]
Unable to read the text as it is now, text color should be 686274 (Gray 100)
